documentation/Migration/Update-1.0.0-rc1.md

```diff
@@ -1,15 +1,86 @@
-Preview Version 0.109.\* to 1.0.0 Relase Candidate 1\*.
-This document is divided
-into two sections: [Breaking Changes](#breaking-changes) and 
-[Suggested Changes](#suggested-changes).
+Preview Version 0.109.\* to 1.0.0 Relase Candidate 1.
+
+Table of Contents
+- [Abstract and Table of Contents](#abstract-and-table-of-contents)
+- [Conceptual Changes](#conceptual-changes)
+  - [Table Discovery](#table-discovery)
+  - [Internal Tables](#internal-tables)
+  - [Metadata Tables](#metadata-tables)
+- [Breaking Changes](#breaking-changes)
+  - [Renamed Classes, Interfaces and Static Methods](#renamed-classes-interfaces-and-static-methods)
+  - [Obsolete Members](#obsolete-members)
+    - [DataSourceInfo](#datasourceinfo)
+    - [DataCookerPath](#datacookerpath)
+    - [DataOutputPath](#dataoutputpath)
+    - [CustomDataProcessor](#customdataprocessor)
+    - [RequiresCookerAttribute](#requirescookerattribute)
+    - [TableConfiguration](#tableconfiguration)
+    - [CustomDataSourceBase](#customdatasourcebase)
+    - [CustomDataSourceAttribute](#customdatasourceattribute)
+    - [CustomDataSourceInfo](#customdatasourceinfo)
+  - [ProcessingSource](#processingsource)
+  - [ColumnRole](#columnrole)
+  - [TableConfiguration](#tableconfiguration-1)
+  - [Data Processors (NOT CustomDataProcessors)](#data-processors-not-customdataprocessors)
+  - [ProcessingSourceInfo](#processingsourceinfo)
+  - [ProcessingSourceAttribute](#processingsourceattribute)
+  - [CustomDataProcessor](#customdataprocessor-1)
+  - [CustomDataProcessorWithSourceParser](#customdataprocessorwithsourceparser)
+  - [TableDescriptor](#tabledescriptor)
+  - [Engine](#engine)
+    - [Creation](#creation)
+  - [Interfaces implemented in the SDK or SDK.Runtime](#interfaces-implemented-in-the-sdk-or-sdkruntime)
+    - [ICustomDataProcessor](#icustomdataprocessor)
+    - [IDataProcessorExtensibilitySupport](#idataprocessorextensibilitysupport)
+    - [IDataExtensionRetrievalFactory](#idataextensionretrievalfactory)
+    - [IMetadataTableBuilderFactory](#imetadatatablebuilderfactory)
+    - [IProcessorEnvironment](#iprocessorenvironment)
+    - [IApplicationEnvironment](#iapplicationenvironment)
+  - [TableAttribute and InternalTableAttribute](#tableattribute-and-internaltableattribute)
+    - [Engine Execution Results](#engine-execution-results)
+  - [IViewportSensitiveProjection](#iviewportsensitiveprojection)
+  - [ViewportSensitiveProjection](#viewportsensitiveprojection)
+  - [IVisibleTableRegion](#ivisibletableregion)
+  - [Plugin Configurations](#plugin-configurations)
+- [Suggested Changes](#suggested-changes)
+  - [Metadata Tables](#metadata-tables-1)
+
+# Conceptual Changes
+
+## Table Discovery
+
+Prior to this release candidate, any tables defined in a processing source's assembly were associated with
+that processing source. The SDK runtime ensured that tables that do not provide a static build method are built
```

Comment:
maybe "The SDK base classes ensured that tables..."? I don't think it was actually the "runtime".

Reply:
I actually think nothing was enforcing this... at most the Engine was

Reply:
maybe I can say "The SDK Engine (and most SDK drivers)"?

Reply:
Maybe don't even mention it. Just note the transition from additional tables to the provider?